build.psm1

```diff
@@ -2765,13 +2775,20 @@ function Start-CrossGen {
     # the NI PS assemblies, we need to replace the IL PS assemblies with the corresponding NI
     # PS assemblies, but with the same IL assembly names.
     #
-    Write-Verbose "PowerShell Ngen assemblies have been generated. Deploying ..." -Verbose
-    foreach ($assemblyName in $psCoreAssemblyList) {
+    Write-Verbose "PowerShell + Add-Type Ngen assemblies have been generated. Deploying ..." -Verbose
```

Comment:
The function Start-Crossgen has [CmdletBinding()] so we should remove the -Verbose.